.github/workflows/comment_bot.yml

```diff
@@ -174,3 +174,18 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           args: "--force"
+
+  issue_assign:
+    name: "Assign issue"
+    permissions:
+      issues: write
+    if: github.event.comment.body == 'take'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
We can remove this checkout by making the .js file into an action, that would be more idiomatic for gha.

Reply:
but I can work on that as a follow up too :)

Reply:
Thanks for the comment, it made me realise I could simplify it a lot as when I started I thought I might need more code for it. The action is not needed though.